Design.py

```diff
@@ -191,61 +193,6 @@ def checkStringForNumber(self, string):
                     return False
```

Comment:
As you said, it should be ``if chr not in numbers``. Anyway, you can replace it with something like this:

``if len([chr for chr in chrs if chr not in numbers]) > 0:
    return False``